src/modules/remote-config/types.ts

```diff
@@ -3,4 +3,9 @@ import type { WalletNameFlag } from 'src/shared/types/WalletNameFlag';
 export interface RemoteConfig {
   extension_wallet_name_flags: Record<string, WalletNameFlag[]>;
   extension_uninstall_link: string;
+  ios_loyalty_config: Partial<{
```

Comment:
Wouldn't it make more sense to use `extension_loyalty_config` instead (that I created recently)? I don't know what is the point of having separate configs for loyalty though...

Reply:
Yeah not sure, I think though this is a temporary location for this value and it'll be moved to the tb api